pkg/apis/seedmanagement/v1alpha1/types_managedseed.go

```diff
@@ -63,8 +63,7 @@ type ManagedSeedSpec struct {
 
 	// Gardenlet specifies that the ManagedSeed controller should deploy a gardenlet into the cluster
 	// with the given deployment parameters and GardenletConfiguration.
-	// +optional
-	Gardenlet *GardenletConfig `json:"gardenlet,omitempty" protobuf:"bytes,3,opt,name=gardenlet"`
+	Gardenlet GardenletConfig `json:"gardenlet,omitempty" protobuf:"bytes,3,opt,name=gardenlet"`
```

Comment:
As the field is no longer optional, we should remove the `omitempty` tag.

```suggestion
	Gardenlet GardenletConfig `json:"gardenlet" protobuf:"bytes,3,opt,name=gardenlet"`
```

Please run `make generate` after this change.